src/openloco/windows/build_vehicle.cpp

```diff
@@ -785,13 +758,18 @@ namespace openloco::ui::build_vehicle
         audio::play_sound(audio::sound_id::click_down, loc16{ x, y, static_cast<int16_t>(pan) }, pan);
         auto item = window->row_info[scrollItem];
         auto vehicleObj = objectmgr::get<vehicle_object>(item);
-        _common_format_args[5] = vehicleObj->name;
+        FormatArguments args{};
+        // Skip 5 * 2 bytes
+        args.push<int32_t>(0);
```

Comment:
Perhaps this is a good time to introduce a skip function to the FormatArgs class. Something like `skip(std::size_t num)`?